### PR TITLE
feat: Support windows build for py/client-ticking

### PR DESCRIPTION
**Pull request by jcferretti**
All tests for `py/client` and `py/client-ticking` passing on win32.
Note "win32" is the name of the Windows API, which is irrespective of 32 bit or 64 bit machine/implementation.
https://stackoverflow.com/questions/61776207/where-does-win32-come-from-when-im-using-windows-64bit

Just as for Excel, we are building in windows for 64 bits, and have not tried 32 bits builds.
Which btw is the same thing that we do in linux, we have not built C++ or python ticking for 32 bit linux, ever.
And we are not doing that for windows (win32) either.

### CODE REVIEW

**Comment by jcferretti** [comment]
capture_005_13102024_223826
capture_004_13102024_223804
capture_003_13102024_223106

**Comment by jcferretti** [comment]
Turns out you can write non-portable code in pure python.

capture_006_13102024_230021

**Comment by jcferretti** [comment]
Capture_554

**Comment by jcferretti** [comment]
All tests passing now, for both `py/client` and `py/client-ticking`.
capture_001_16102024_205124

**Review by jmao-denver — COMMENTED** [review]
The changes LGTM. I think we want to add the Windows build/publishing it to the GH CI?

**Comment by jcferretti** [comment]
> The changes LGTM. I think we want to add the Windows build/publishing it to the GH CI?

That would be interesting and desirable, but setting up windows CI is beyond the scope of this humble PR.  One step at a time.

File: py/client-ticking/setup.py [hunk 1]
```diff
@@ -29,6 +30,27 @@ def _compute_version():
     return _normalize_version(os.environ['DEEPHAVEN_VERSION'])
 
 _version = _compute_version()
+_system = platform.system()
+
+if _system == 'Windows':
+    dhinstall=os.environ.get('DHINSTALL')
```
Comment (jmao-denver) [inline py/client-ticking/setup.py:36]: Should we warn if `dhinstall` is None or empty?

Comment (jcferretti) [reply py/client-ticking/setup.py]: Good catch.  I think just failing (by moving from `.get` to `[]` to access the environ dict) is better, also more consistent with what the script was already doing for other environment variables.

**Review by jmao-denver — APPROVED** [review]
LGTM

**Pull request merged**